packages/storybook/src/templates/componenten/index.tsx

```diff
@@ -97,7 +108,20 @@ export default function Componenten() {
                       <UnorderedList>
                         {frameworkOptions.map((option) => (
                           <UnorderedListItem key={option}>
-                            <FormFieldCheckboxOption label={option} onChange={() => handleFrameworkChange(option)} />
+                            <FormFieldCheckboxOption
+                              label={
+                                <>
+                                  {option}
+                                  <NumberBadge
+                                    aria-label={`${frameworkCounts[option]} ${frameworkCounts[option] === 1 ? 'component' : 'componenten'}`}
```

Comment:
Zou je als aria label nog de spatie en haakjes willen toevoegen, zodat het geheel leest als `Label (x componenten)`? Dat helpt bij de intonatie van screenreaders.

```suggestion
                                    aria-label={`${frameworkCounts[option]} ${frameworkCounts[option] === 1 ? 'component' : 'componenten'}`}
```

Reply:
✅